documentation/modules/ROOT/pages/transformations/swap-geometry-coordinates.adoc

```diff
@@ -0,0 +1,68 @@
+[id="swap-geometry-coordinates"]
+= SwapGeometryCoordinates
+
+:toc:
+:toc-placement: macro
+:linkattrs:
+:icons: font
+:source-highlighter: highlight.js
+
+toc::[]
+
+When serializing geometry information into a WKB (Well-Known Binary) or EWKB (Extended Well-Known Binary) byte array, the source database will choose to use one of two different coordinate systems:
+
+Legacy CRS:84::
+This is an older convention that serializes coordinates using `(x, y)` or `(longitude, lattitude)` axis order.
+
+European Petroleum Survey Group (EPSG)::
+This is a modern convention that serializes coordinates using `(y, x)` or `(lattitude, longitude)` axis order.
+
+The format irrelevant in a {prodname} source connector since the data is written to the change event using the source database's chosen coordinate system.
+However, for a consumer, it's essential to understand the coordinate system used by the source and what is expected at the target.
```

Comment:
```suggestion
However, a sink connector must not only be able to interpret the coordinate system of the source database, but also provide data to the target system in  a format that it can consume.
```